src/components/ChatApp/TopBar.react.js

```diff
@@ -44,17 +50,23 @@ class TopBar extends Component {
 					}
 					targetOrigin={{ horizontal: 'right', vertical: 'top' }}
 					anchorOrigin={{ horizontal: 'right', vertical: 'top' }}
-					>
+				>
 					<MenuItem primaryText="Settings"
-						onClick={this.props.handleSettings} />
+					onClick={this.props.handleSettings}
+					rightIcon={<Settings/>}/>
 					<MenuItem primaryText="Custom Theme"
 						key="custom"
-						onClick={this.props.handleThemeChanger} />
-					<MenuItem primaryText="Change Password" onTouchTap={this.props.handleChangePassword} />
+					onClick={this.props.handleThemeChanger}
+					rightIcon={<Edit/>}/>
+					<MenuItem primaryText="Change Password"
+					onTouchTap={this.props.handleChangePassword}
```

Comment:
![er](https://user-images.githubusercontent.com/13276887/28104053-c385ae62-66f6-11e7-852b-a3a2dbf3d09d.png)

Sorry, I missed it in the code. But its not showing in the demo link. Can you please check once?